webdriver-spec.html

```diff
@@ -1211,9 +1211,9 @@
  </tr>
 
  <tr>
-  <td><dfn>invalid element coordinates</dfn>
+  <td><dfn>invalid coordinates</dfn>
   <td>400
-  <td><code>invalid element coordinates</code>
+  <td><code>invalid coordinates</code>
```

Comment:
Hmm. There’s going to be a lot of backwards compatibility work to make this change, but I guess it’s the right thing to do.